File: cpp/zadanie6.cpp
#include <iostream>
#include <fstream>
#include <string>

int main() {
	std::fstream inFile;
	std::fstream outFile;

	inFile.open("dane.txt", std::ios::in);
	outFile.open("wynik.txt", std::ios::out);
	
	std::string date;
	int ile = 0;

	while (!inFile.eof()) {
		date = "";
		inFile >> date;
		if (date == "May")
			ile++;
			
		outFile << ile << std::endl; 
	}

	inFile.close();
	outFile.close();

	return 0;
}

File: javascript/zadanie6.js
var fs = require('fs')

var file = fs.readFileSync('dane.txt').toString('utf8')
var dates = file.split('\r\n')
var ile

if (fs.existsSync('wynik.txt'))
    fs.unlinkSync('wynik.txt')

dates.forEach(date => {
    if (String(date) === "May")
        ile++
})

fs.appendFileSync('wynik.txt', ile + '\r\n')